src/AppBundle/Controller/Api/SurveyController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Entity\SurveyAnswer;
+use AppBundle\Entity\Survey;
+use AppBundle\Entity\SurveyQuestion;
+use Mcfedr\JsonFormBundle\Controller\JsonController;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\Request;
+
+class SurveyController extends JsonController
+{
+    /**
+     * @Route("/surveys", name="api_surveys")
+     * @Method("GET")
+     */
+    public function apiSurveysAction()
+    {
+        $user = $this->getUser();
+        if (!$user) {
+            return $this->json(['message' => 'User is not authorized'], 401);
+        }
+        $em = $this->getDoctrine()->getManager();
+        $surveys = $em->getRepository(Survey::class)->findSurveyByUser($user);
+        if (!$surveys) {
```

Comment:
If you didn`t surveys for user, you will send a successful response with empty array.